src/components/calcite-link/calcite-link.scss

```diff
@@ -46,42 +46,40 @@
 
 // disabled styles
 :host([disabled]) {
-  pointer-events: none;
+  @apply pointer-events-none;
   span,
   a {
-    pointer-events: none;
+    @apply pointer-events-none;
     opacity: var(--calcite-ui-opacity-disabled);
   }
 }
 // icon positioning and styles
 :host .calcite-link--icon.icon-start {
-  margin-right: $baseline/3;
+  @apply mr-2;
 }
 
 :host([dir="rtl"]) .calcite-link--icon.icon-start {
-  margin-right: 0;
-  margin-left: $baseline/3;
+  @apply mr-0 ml-2;
 }
 
 :host .calcite-link--icon.icon-end {
-  margin-left: $baseline/3;
+  @apply ml-2;
 }
 
 :host([dir="rtl"]) .calcite-link--icon.icon-end {
-  margin-left: 0;
-  margin-right: $baseline/3;
+  @apply ml-0 mr-2;
 }
 
 // inline
 @mixin link-style($color, $color-hover, $border-color) {
```

Comment:
It would be better to get rid of this mixin all together. It is only used once and the redirection of css-vars to sass vars means you can't use the tailwind helpers. Just move these styles to the actual place they're used and delete this.

Reply:
updated.

Reply:
Any thoughts here?